projects/laji/src/app/+taxonomy/taxon/info-card/shared/component/taxon-bold/taxon-bold.component.scss

```diff
@@ -0,0 +1,3 @@
+.bold-entry {
```

Comment:
Just a note, we have utility classes for margins (for example `mb-1` gives small bottom margin) so those could be used instead of creating new classes

Reply:
Thanks for the tip, tried to utilize mb-1 and to remove the unnecessary new styles now.